.sbtopts

```diff
@@ -1 +1 @@
--J-XX:MaxMetaspaceSize=512M -Dsbt.override.build.repos=true -Dsbt.repository.config=project/repositories
```

Comment:
This line should be kept intact. I have made some changes to Nexus configs on repo.icos-cp.eu, so it should hopefully work now. We need to rely on it in any case, because that is where our own SBT plugins are published.

Reply:
I think it should work either way. If we keep the override, our nexus repo acts as a proxy and would use the upstream repo if its layout policy is set up as permissive. If we remove the override, the default repos and ours would be used. I can change it back, though.